terraform/modules/logging/alarms/missing-logs/alarms.tf

```diff
@@ -3,16 +3,16 @@
 // The alarm will be triggered when the metric is not emitted due to the breaching poicy on missing data
 
 resource "aws_cloudwatch_metric_alarm" "missing_logs_alarm" {
-  count             = "${length(var.app_names)}"
-  alarm_name        = "${var.environment}-missing-logs-${var.app_names[count.index]}"
+  count             = length(var.app_names)
+  alarm_name        = var.environment}-missing-logs-${var.app_names[count.index]
```

Comment:
Some extra curly braces snuck in here

Reply:
Oh dear. I'd recommend not reviewing further - probably more efficient for us to fix all of these first.